spec/features/user_show_spec.rb

```diff
@@ -26,6 +26,18 @@
     expect(page).to have_content('Post #2')
     expect(page).to have_content('Post #3')
   end
+
+  scenario 'Redirects to the post show page when a post is clicked' do
+    visit user_path(user)
+
+    click_link post2.title
+    expect(current_path).to eq(user_post_path(user_id: post2.author_id, id: post2.id))
+  end 
+
+  scenario 'clicking on See All Posts redirects to the user\'s post index page' do
+    click_link 'See All Posts'
+    expect(page).to have_current_path(user_posts_path(user))
+  end
```

Comment:
Nice job so far, you still haven't met the requirements as pointed out by the previous reviewers. Just as they have pointed out, in the `user_show_spec.rb` add a test that asserts you can see the number of posts the user has written. This is a requirement to move to the next milestone, kindly add it to avoid blockers👍🏼.
<img width="555" alt="image" src="https://github.com/noelfoka/Blog_app/assets/64643995/17ae541e-67fb-4d78-a2e1-198c982bf96a">

Reply:
Can you help me to understand more

Reply:
Because all my test pass in my computer

Reply:
Hello @noelfoka , If all your tests are passing, kindly add the test case pointed out above👍🏼. If you still have questions, You can reach out on slack for faster response👍🏼.